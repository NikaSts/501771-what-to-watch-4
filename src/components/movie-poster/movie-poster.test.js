import React from 'react';
import renderer from 'react-test-renderer';
import MoviePoster from './movie-poster';


const title = `New Year`;
const image = `the-grand-budapest-hotel`;

it(`MoviePoster should render correctly if user is logged in`, () => {
  const tree = renderer
  .create(
      <MoviePoster
        title={title}
        image={image}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
