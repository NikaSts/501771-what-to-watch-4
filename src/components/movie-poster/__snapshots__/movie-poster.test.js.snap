// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviePoster should render correctly if user is logged in 1`] = `
<div
  className="movie-card__poster"
>
  <img
    alt="New Year"
    height="327"
    src="img/the-grand-budapest-hotel-poster.jpg"
    width="218"
  />
</div>
`;
